x-pack/plugins/security_solution/public/timelines/components/timeline/body/actions/add_note_icon_item.tsx

```diff
@@ -24,20 +25,24 @@ const AddEventNoteActionComponent: React.FC<AddEventNoteActionProps> = ({
   showNotes,
   timelineType,
   toggleShowNotes,
-}) => (
-  <ActionIconItem>
-    <NotesButton
-      ariaLabel={ariaLabel}
-      data-test-subj="add-note"
-      showNotes={showNotes}
-      timelineType={timelineType}
-      toggleShowNotes={toggleShowNotes}
-      toolTip={
-        timelineType === TimelineType.template ? i18n.NOTES_DISABLE_TOOLTIP : i18n.NOTES_TOOLTIP
-      }
-    />
-  </ActionIconItem>
-);
+}) => {
+  const { kibanaSecuritySolutionsPrivileges } = useUserPrivileges();
+  return (
+    <ActionIconItem>
+      <NotesButton
+        ariaLabel={ariaLabel}
+        data-test-subj="add-note"
+        isDisabled={kibanaSecuritySolutionsPrivileges.crud === false}
```

Comment:
Consider adding this check to the `Pin event` button, to avoid the behavior shown in the video below:

https://user-images.githubusercontent.com/4459398/173874849-b2e54970-c55f-47ce-9132-d015f4d57439.mov

Reply:
Consider adding this check to the `Add note` button in the `Notes` tab, to avoid the behavior shown in the video below:

https://user-images.githubusercontent.com/4459398/173875209-0ad09e30-622f-4b88-9b6f-54dddb7d37e2.mov